bundle-workflow/src/test_workflow/perf_test_cluster.py

```diff
@@ -0,0 +1,78 @@
+import json
+import os
+import subprocess
+from contextlib import contextmanager
+
+from test_workflow.test_cluster import TestCluster
+
+
+class Cluster:
+    @contextmanager
+    def create(manifest, config, stack_name, security):
+        perf_test_cluster = PerfTestCluster(manifest, config, stack_name, security)
+        try:
+            perf_test_cluster.create()
+            yield perf_test_cluster.endpoint()
+        finally:
+            perf_test_cluster.destroy()
+
+
+class PerfTestCluster(TestCluster):
+    """
+    Represents a performance test cluster. This class deploys the opensearch bundle with CDK and returns the private IP.
+    """
+
+    def __init__(self, bundle_manifest, config, stack_name, security):
+        self.manifest = bundle_manifest
+        self.security_id = config['Constants']['SecurityGroupId']
+        self.vpc_id = config['Constants']['VpcId']
+        self.account_id = config['Constants']['AccountId']
+        self.region = config['Constants']['Region']
+        self.role = config['Constants']['Role']
+        self.work_dir = 'tools/cdk/mensor/single-node/'
+        self.stack_name = stack_name
+        self.cluster_endpoint = None
+        self.cluster_port = None
+        self.output_file = 'output.json'
+        self.ip_address = None
+        self.security = 'enable' if security else 'disable'
+        self.params_dict = {
```

Comment:
Since this is not used elsewhere should just be a local var. Same for others.

Reply:
Created local variables